lekko_client/evaluation/rules.py

```diff
@@ -24,60 +24,58 @@ def evaluate_rule(rule: Rule, namespace: str, config_name: str, context: ClientC
     if not rule_type:
         raise EvaluationError("Empty rule")
 
-    rule_value = getattr(rule, rule_type)
-
     if rule_type == "bool_const":
-        return rule_value
+        return rule.bool_const
     elif rule_type == "not":
+        rule_value = getattr(rule, rule_type)
```

Comment:
we have to use `getattr` here because `not` is a reserved word in python, oops :)